R/class-workbook.R

```diff
@@ -854,6 +858,7 @@ wbWorkbook <- R6::R6Class(
     add_data_table = function(
         sheet       = current_sheet(),
         x,
+        dims        = NULL,
         startCol    = 1,
         startRow    = 1,
         xy          = NULL,
```

Comment:
What are we doing with `xy`?  Is this doing to be deprecated in favor of `dims`?

Reply:
good thinking. I never had a use for `xy`. Guess we should deprecate it.